heroic-core/src/main/java/com/spotify/heroic/http/tracing/OpenCensusApplicationEventListener.java

```diff
@@ -278,6 +279,7 @@ private void log(OpenCensusFeature.Verbosity level, String s) {
         private void logError(final Throwable t) {
             requestSpan.setStatus(Status.INTERNAL);
             requestSpan.putAttribute("event", AttributeValue.stringAttributeValue("error"));
+            requestSpan.putAttributes(environmentMetadata.toAttributes());
```

Comment:
Having the metadata here will only attach it when a request throws an error. You should move it to the span start so its on all of them - probably the request span, not the resource span, since the request span will always be created.